libraries/api-extractor/src/DocElementParser.ts

```diff
@@ -195,7 +197,8 @@ export default class DocElementParser {
     if (linkDocElement && pipeSplitContent.length > 1) {
       const displayTextParts: string[] = pipeSplitContent[1].match(this._wordRegEx);
       if (displayTextParts && displayTextParts[0].length !== pipeSplitContent[1].length) {
```

Comment:
We discussed it.  I will address it in a separate PR

---
In reply to: [122555107](https://github.com/Microsoft/web-build-tools/pull/239#discussion_r122555107) [](ancestors = 122555107)